playbooks/roles/foreman_installer/tasks/packages.yml

```diff
@@ -8,7 +8,6 @@
 - name: 'Install foreman-release-scl'
   yum:
     name: foreman-release-scl
-    update_cache: yes
```

Comment:
Given `foreman-release-scl` lays down new repositories, are we positive an update to the cache isn't needed to ensure yum picks up the new repositories?